docssrc/src/commandtrees.md

```diff
@@ -80,6 +80,108 @@ In this example, we have our normal `/sayhi` command using the `executes()` meth
 
 </div>
 
+### uses nested arguments to reduce indentation levels
```

Comment:
I'd recommend capitalizing the first letter of the first word in the heading and each of the sentences added in this section.

Perhaps also reword this heading?

```suggestion
### Reduce indentation with nested argument
```